src/main/java/com/hotjava/app/hotjava/dao/VehicleSQLDAO.java

```diff
@@ -9,7 +9,6 @@
 import java.util.List;
 import java.util.Optional;
 
-//@Repository("vehicleDAO")
```

Comment:
If this repository when left uncommented has functionality, place in a separate database branch so that you can keep files your teammates are working on readable.